.github/rundecks/GISS_ONLY.R

```diff
@@ -232,8 +232,8 @@ aer_rad_forc=0   ! turn off aerosol radiative forcing diagnostics
 cloud_rad_forc=1 ! turn on cloud radiative forcing diagnostics
 
 ! diagnostics
-! SUBDD='OH:4 NO:4 O3:4 NO2:4 CO:4 CH4:4 PS:4' ! TODO: Turn these on
-SUBDD=''
+SUBDD='OH:4 NO:4 O3:4 NO2:4 CO:4 CH4:4 PS:4'
+! SUBDD=''
```

Comment:
This comment is also redundant.
```suggestion
SUBDD='OH:4 NO:4 O3:4 NO2:4 CO:4 CH4:4 PS:4'
```